src/mount/sauna_client.cc

```diff
@@ -267,12 +268,12 @@ void convert_winfsp_context_to_master_context(uint32_t &uid, uint32_t &gid) {
 }
 
 void patch_uid_gid_fields(stat32 &st) {
-	if (((st.st_uid ==
-	      (mounting_uid == USE_LOCAL_ID ? winfsp_uid : mounting_uid)) ||
-	     (st.st_mode & S_IWOTH) ||
-	     (st.st_gid ==
-	          (mounting_gid == USE_LOCAL_ID ? winfsp_gid : mounting_gid) &&
-	      (st.st_mode & S_IWGRP)))) {
+	if ((st.st_uid ==
+	     (mounting_uid == USE_LOCAL_ID ? winfsp_uid : mounting_uid)) ||
+	    (st.st_mode & S_IWOTH) ||
+	    (st.st_gid ==
+	         (mounting_gid == USE_LOCAL_ID ? winfsp_gid : mounting_gid) &&
+	     (st.st_mode & S_IWGRP))) {
```

Comment:
This is not necessary for the PR (it is just removing an extra pair of parenthesis), but it is not so unrelated.

Reply:
I noticed :sweat_smile: